packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -233,12 +234,18 @@ class App extends client.App {
               client.composeMessage(error.message)
             ),
             (nodeIdsMap) => {
-              this.props.actions.startDebuggerSession(
-                createSystemMessage('Debug session started'),
-                nodeIdsMap,
+              Maybe.maybe(
+                null,
+                (currentPatchPath) => {
```

Comment:
Side-effect maybe. I think `isNothing` + `explode` will be more appropriate here.